pkg/json_raw_results.go

```diff
@@ -609,11 +609,9 @@ func (r *jsonScorecardRawResult) addBinaryArtifactRawResults(ba *checker.BinaryA
 //nolint:unparam
 func (r *jsonScorecardRawResult) addSecurityPolicyRawResults(sp *checker.SecurityPolicyData) error {
 	r.Results.SecurityPolicies = []jsonFile{}
-	for _, v := range sp.Files {
-		r.Results.SecurityPolicies = append(r.Results.SecurityPolicies, jsonFile{
-			Path: v.Path,
-		})
-	}
+	r.Results.SecurityPolicies = append(r.Results.SecurityPolicies, jsonFile{
```

Comment:
Ideally information about SecurityPolicyInformationTypeEmail, etc should be reported here. This way someone who fetches results from the REST API (once the raw results are exposed) will be able to parse and create their own policies based on what kind of information was found.
One suggestion it to create a `jsonSecurityFile`:
```golang
type jsonSecurityFile struct{
  jsonFile
  Type string --> this will take a string like SecurityPolicyInformationTypeEmail, etc
}
```

Later we can "fix" the jsonFile structure to contain a list of Snippet/lines. @raghavkaul fyi

Reply:
closing this comment for now - we can revisit if necessary

Reply:
tell me what the confusion is. I can try to explain or we can chat offline

Reply:
I would like to chat... I simply don't have my head into json raw results - my sense looking here is that I could create a pattern for security policy hits that follows TokenPermissions or DangerousWorkFlow or DependencyPinning which appears to emit Snippet/offset/file information -- but in this case to would be SecurityInformationType/Match/linenumber/File information in a `jsonSecurityFile`

If I got that right - great, I can do that, if I am still missing the point - we should chat

Reply:
after studying raw results - I now understand and have changed the raw results to emit all the findings from the security policy check.